bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/VoiceManager.java

```diff
@@ -106,11 +106,11 @@ public interface VoiceManager {
      * Interprets the passed string using a particular HLI service and the default locale.
```

Comment:
Please update the description to explain how is handled the comma separated list of interpreters.